src/index.js

```diff
@@ -49,7 +49,7 @@ export default function (CustomElement, opts) {
     }
     componentWillReceiveProps(props) {
       const node = ReactDOM.findDOMNode(this);
-      Object.keys(props).forEach(name => {
+      for (var name in props) {
         if (name === 'children' || name === 'style') {
           return;
```

Comment:
This is prevent the build from passing. This will force the entire method to `return` when you want to `continue` because you've changed the way you're looping.